pandunia-loge.csv

```diff
@@ -225,6 +223,8 @@ avar.an:i|avarani||||harmful (damaging)||dañino (perjudicial)|||||||||||||dama
 Axur:|Axur|xefocite|||Assur||Assur (Ashur)||||||||||||||Assur|Aszur
 Axur:ia|Axuria|||akk:(aššur), ara:(aššur), hin:अश्शूर (aśśūr), fas:(āšūr)|Assyria||Asiria||||||||||||||Assyria|Asyria
 ay:|ay|||spa:ay, por:ai, fra:aïe, zho:哎 (ai), kor:아야 (aya), fas:(ây), rus:ай (ay)|ouch (ow, alas)|aïe!|ay!|ai!||||||||||||aj (alas)|ai! (au!)|aj!, ał!
+az:i|azi||||Asian|asiatique|asiático|||||||||||||azia|aasialainen|azjatycki
+Az:ia|Azia|||eng:spa:Asia, rus:Азия (Aziya), tur:Asya, ara:آسيا (ʾāsiyā), fas:آسیا‎ (âsiyâ), hin:एशिया (eśiyā), ben:এশিয়া (eśiẏa), mar:आशिया (āśiyā), may:Asia, zho:亞洲 (yàzhōu), yue:亚洲 (aa3 zau1), wuu:亚洲 (ia tseu), jpn:アジア (ajia), kor:아시아 (asia)|Asia|Asie|Asia|||||||||||||Azio|Aasia|Azja
```

Comment:
Voiceless /s/ is more common in Asian languages like Turkish, Arabic, Persian, Hindi, Bengali and Malay, so it's better to keep "Asia", which also looks more familiar to most.